### ISSUE TITLE
Link for email list signup on dib-training website needs updating

### ISSUE THREAD

**Issue by johnsolk** [ged-lab/dib-training#38]
The email signup on link points to former @lists.idyll.org

http://dib-training.readthedocs.io/en/pub/

Is this the right link to point to? 

https://dib-lab.groups.io/g/training

**Comment by charlesreid1** [comment]
[link to relevant markdown file](https://github.com/dib-lab/dib-training/blob/pub/index.rst), link is near top

**Comment by ctb** [comment]
note that I think groups.io provides html forms for embedding.

**Comment by charlesreid1** [comment]
Yes - you need to be an admin of the group. It's in Admin > Promote, there's some HTML you can copy and paste.
